tests/Exercism.TestRunner.FSharp.IntegrationTests/Solutions/MultipleTestsWithMultipleFails/expected_results.json

```diff
@@ -0,0 +1,19 @@
+{
+  "status": "fail",
+  "tests": [
+    {
+      "name": "FakeTest.Add_should_add_numbers",
+      "status": "fail",
+      "message": "FsUnit.Xunit+MatchException : Exception of type 'FsUnit.Xunit+MatchException' was thrown.\nExpected: Equals 3\nActual:   was 2"
```

Comment:
Maybe we could strip the "FsUnit.Xunit+MatchException : Exception of type 'FsUnit.Xunit+MatchException' was thrown." part from the message too? Sorry that I just noticed this. It's also fine if you'd like to do this in a separate PR.

Reply:
Done. See commit 278b27365a55420d00c3afd2793e8045c4cae97a

Reply:
Superb!